env/index.js

```diff
@@ -102,6 +102,18 @@ class Env {
 
     return env === 'testing'
   }
+
+  /**
+   * Determines whether the `NODE_ENV` environment
+   * variable is set to the given `environment`.
+   *
+   * @returns {Boolean}
+   */
+
+  is (environment) {
+    const currentEnvironment = this.get('NODE_ENV', environment).toLowerCase()
+    return currentEnvironment === environment
```

Comment:
Can you please add an empty line before the return statement?

Reply:
Done :)